index.html

```diff
@@ -11,18 +11,179 @@
+                            contact
+                            me
+                            by
+                            clicking the icons below.
+                        </p>
+                    </div>
+                    <div>
+                        <p class="hero-social">
+                            let's connect
+                        </p>
+                        <ul class="hero-icons d-flex">
+                            <li>
+                                <a target="_blank" href="https://twitter.com/Zuhaib042" rel="noopener noreferrer"><img
+                                        src="./images/twitter.svg" alt="twitter-logo"></a>
+                            </li>
+                            <li>
+                                <a target="_blank" href="https://www.linkedin.com/in/zuhaib-amjad-488914234/"
+                                    rel="noopener noreferrer"><img src="./images/linkedin.svg" alt="linkedIn-logo"></a>
+                            </li>
+                            <li>
+                                <a href="#" rel="noopener noreferrer"><img src="./images/medium.svg"
+                                        alt="medium-logo"></a>
+                            </li>
+                            <li>
+                                <a target="_blank" rel="noopener noreferrer" href="https://github.com/Zuhaib042"><img
+                                        src="./images/github.svg" alt="github-logo"></a>
+                            </li>
+                            <li>
+                                <a href="#" rel="noopener noreferrer"><img src="./images/good.svg"
+                                        alt="angellist-logo"></a>
+                            </li>
+                        </ul>
+                    </div>
+                </div>
+            </div>
+        </section>
+
+        <!-- works section -->
+
+        <section class="works">
+            <h1 class="works-title">My Projects</h1>
+            <article class="d-flex flex-column card">
+                <div class="card-image">
+                    <img src="./images/portfolio-img4.jpg" alt="example-image1">
+                </div>
+                <div class="card-content">
+                    <div class="primary-text">
+                        <h2>Tonic</h2>
+                        <div class="frame d-flex">
+                            <h6>CANOPY</h6>
+                            <div class="circle"></div>
+                            <p>Back End Dev</p>
+                            <div class="circle"></div>
+                            <p>2015</p>
+                        </div>
+                        <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+                    </div>
+                    <ul class="tags d-flex list-none">
+                        <li>html</li>
```

Comment:
- [ ] Please try to stick to figma design as much as possible to make your project looks more professional . you can do so by trying by matching like colors, font-size ,font-weight etc.

Figma design:
![image](https://user-images.githubusercontent.com/90524466/185684813-4f222d34-be68-4864-a10b-e1e8296db9d4.png)

Your design:
![image](https://user-images.githubusercontent.com/90524466/185684913-d0a62de4-e302-4567-adab-ec697b4ef156.png)

Reply:
got it. i have fixed it now